.env-starter

```diff
@@ -0,0 +1,49 @@
+FB_WEB_API_KEY=
+FB_WEB_AUTH_DOMAIN=
+FB_WEB_DB_URL=
+FB_WEB_PROJECT_ID=
+FB_WEB_STORAGE_BUCKET=
+FB_WEB_MSG_SENDER_ID=
+FB_WEB_APP_ID=
+FB_WEB_MEASUREMENT_ID=
+
+FB_IOS_APP_ID=
+FB_IOS_API_KEY=
+FB_ANDROID_APP_ID=
+FB_ANDROID_API_KEY=
+FB_ANDROID_CERT_HASH=
+
+FB_HOSTING_DASHBOARD_URL=
+FB_HOSTING_WEB_URL=
+
+GOOGLE_CLIENT_ID_NATIVE=...
+GOOGLE_CLIENT_ID_ANDROID=
+GOOGLE_CLIENT_ID_IOS=
+
+SENTRY_DSN=
+SENTRY_URL=
+SENTRY_ORG=
+SENTRY_PROJECT=
+SENTRY_AUTH_TOKEN=
+
+APP_BUNDLE_ID=
+
+IOS_DEVELOPMENT_TEAM=
+IOS_REVERSED_CLIENT_ID=
+
+EMAIL_PROJECT_NAME=
+EMAIL_PROJECT_NAME_PRODUCTION=
+EMAIL_SENDGRID_API_KEY=
+EMAIL_SENDGRID_TEMPLATE_ID=
+EMAIL_SENDGRID_VERIFICATION_TEMPLATE_ID=
+EMAIL_FROM_ADDRESS=
+EMAIL_ADMIN_EMAIL=
+
+DASHBOARD_CLIENT_INVITATION_LINK=
+DASHBOARD_URL=
```

Comment:
Is this the field that will store our future production dashboard url rather than the fb hosted one?

Reply:
Right now it's just the URL that is provided by Firebase, but we can definitely re-use this variable down the road.